test/cypress/plugins/stubs.js

```diff
@@ -535,7 +535,29 @@ module.exports = {
             headers: {
               'Accept': 'application/json'
             },
-            body: goLiveRequestFixtures.validPostGovUkPayAgreementRequest(opts)
+            body: goLiveRequestFixtures.validPostGovUkPayAgreementRequest(opts).getPlain()
+          }
+        }],
+        responses: [{
+          is: {
+            statusCode: 201,
+            headers: {}
+          }
+        }]
+      }
+    ]
+  },
+  postStripeAgreementIpAddress: (opts) => {
+    return [
+      {
+        predicates: [{
+          equals: {
+            method: 'POST',
+            path: `v1/api/services/${opts.external_id}/stripe-agreementt`,
```

Comment:
There's an extra "t" in the path, do the tests pass?

Reply:
Yes, they do 🤔